### PR TITLE
Fix small typo: is updates --> is updated

### PR DESCRIPTION
**Pull request by wasosa**
Fix small typo: is updates --> is updated

### CODE REVIEW

**Comment by irvifa** [comment]
@wasosa please sign the CLA

**Comment by wasosa** [comment]
Sure thing. Done.
I signed it.

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -752,7 +752,7 @@ apply multiple fixes in between pausing and resuming without triggering unnecess
     REVISION  CHANGE-CAUSE
     1   <none>
     ```
-* Get the rollout status to ensure that the Deployment is updates successfully:
+* Get the rollout status to ensure that the Deployment is updated successfully:
```
Comment (kbhawkey) [inline content/en/docs/concepts/workloads/controllers/deployment.md:755]: @wasosa . Thanks for your contribution.
/lgtm

**Pull request merged**